spec/ParseGeoPoint.spec.js

```diff
@@ -24,7 +24,7 @@ describe('Parse.GeoPoint testing', () => {
         });
       }
     });
-  });
+  }, 60000);
```

Comment:
I'll remove them, those were to bypass the failures that we had for a while in the geo point tests.